src/styles/community-page.scss

```diff
@@ -0,0 +1,73 @@
+.container {
+  display: grid;
```

Comment:
CSS Grid has limited support as of 2020. Does this design has a fallback that works in older browsers? This is more of a question. What's the ideal browser usage we support here?